### ISSUE TITLE
License Agreement says "before May 8th 2014"

### ISSUE THREAD

**Issue by goodtaster** [CounterpartyXCP/counterwallet#640]
"If you are trying to access a wallet created _before_ May 8, 2014, you have typed the passphrase incorrectly."

I'm guessing this should be "after".

**Comment by rippler** [comment]
* I can't find the source and its history on Github right now, but I think it used to say "old wallet" and I changed that to "wallet created before May 8, 2014".
* I think people who try to access such wallets should prefix their real (old) pass phrase with "old ", so this still seems correct to me. Although I don't have a first-hand experience with old wallets.

**Comment by goodtaster** [comment]
It is incorrect:

"If you are being asked to accept the Terms and Conditions, this may be the first time you are opening this wallet on this server. If you are trying to access a wallet created before May 8, 2014, you have typed the passphrase incorrectly."

If they have a wallet created before May 8, it will not be accessible even if the passphrase is correct. (As you said that has to be prefixed by old.) If the wallet is created after May 8, then the only way this screen can pop up is if the passphrase has been typed incorrectly (hence the warning).

**Comment by rippler** [comment]
> If they have a wallet created before May 8, it will not be accessible even if the passphrase is correct.

*The* wallet won't be, but *a* wallet will be. 
What the warning is saying is this:  "If your pass phrase is "like just love know never want time out there make look eye" and you first started using Counterwallet before May 8, 2014, you are logging in to a different wallet from what you expect."

Why? Because in that case you should probably add "old " before that pass phrase.

But the warning appears to anyone, not to only some users who used some particular pass phrase.
It appears even to people who created a completely new address on the same server just 5 seconds ago.

There is no "incorrect" pass phrase, it's the main message we deliver to users confused about pass phrases. Any 12 word combo from the 1600+ word list will do. You can login like this: "like like like like like like like like like like like like". 
Counterwallet can't a priori know if your "12x like" is correct or whether you should be using "old " + "12x like". It just warns anyone and everyone precisely because it can't tell. If you prefix that pass phrase with "old like like like ..." then it will know for sure and use the old method to determine a list of your addresses.

You can edit the string and create a pull request in `develop` branch:
https://github.com/CounterpartyXCP/counterwallet/blob/develop/src/locales/en/translation.json

**Comment by goodtaster** [comment]
True, I guess we shouldn't imply incorrect/corect passphrases. But it's not really clear. And as you said we shouldn't really let people know about adding "old" behind the passphrase since it is a security risk and may confuse them. 

Maybe just say if the passphrase was created before May 8, you should contact support... I figure most people will have switched by now. That way we don't risk confusing anyone.
